test/extensions/filters/http/cdn_loop/parser_corpus/rfc8586-example.txt

```diff
@@ -0,0 +1 @@
+foo123.foocdn.example, barcdn.example; trace="abcdef",AnotherCDN; abc=123; def="456"
```

Comment:
It would be great to add more files that directly line up with the examples in `parser_test`, just for some more seed input.

Reply:
Done.